terraform/aws-accounts/cloud-platform-aws/vpc/eks/cluster.tf

```diff
@@ -103,10 +103,10 @@ locals {
     }
   }
 
-  monitoring_ng_12_12_23 = {
-    desired_size = 2
-    max_size     = 3
-    min_size     = 2
+  monitoring_ng_24_06_24 = {
+    desired_size = 4
```

Comment:
We need to increase this to cover one node per prom instance in each AZ and 1 extra to cover any resources that don't fit on those instances (we expect the Thanos compactor resources to increase).

The prom instances have [node affinity and pod anti-affinity policies to ensure that they are placed in separate AZs](https://github.com/ministryofjustice/cloud-platform-terraform-monitoring/blob/4d8747368d0623e75f8d566ea34eab8252d0751e/templates/prometheus-operator-eks.yaml.tpl#L531)